MonoGame.Framework/Audio/SoundEffect.cs

```diff
@@ -63,7 +71,11 @@ internal SoundEffect(byte[] header, byte[] buffer, int bufferSize, int durationM
 
         // Only used from XACT WaveBank.
         internal SoundEffect(MiniFormatTag codec, byte[] buffer, int channels, int sampleRate, int blockAlignment, int loopStart, int loopLength)
-        {
+        {
+            Initialize();
+            if (_systemState != SoundSystemState.Initialized)
```

Comment:
These should maybe throw?

Reply:
I believe there are not even useful since Initialize() throws.